Themis/ViewModels/CorrectionSidebar/ProblemStatementCellViewModel.swift

```diff
@@ -62,4 +63,12 @@ class ProblemStatementCellViewModel: ObservableObject {
             .replacingOccurrences(of: "[task][", with: "") // to remove symbol
             .replacingOccurrences(of: "](.*())", with: "", options: .regularExpression) // to remove tests
     }
+
+    func replaceTestsColor(_ problemStatement: String) -> String {
+        return problemStatement
+            .replacingOccurrences(of: "testsColor\\([A-z]+\\)", with: "red", options: .regularExpression) // replace test color by red (or green)
```

Comment:
where is the color replaced to green if a test passes?

Reply:
That's part of a next story.

Reply:
0k then I would suggest making the colored parts black as long as this is not working

Reply:
True, i implement it to be in black.